akinet/http/parser_test.go

```diff
@@ -261,6 +261,7 @@ func TestHTTPRequestParser(t *testing.T) {
 	}
 
 	for _, c := range testCases {
+		t.Logf(c.name)
```

Comment:
Yeah, I thought one of the test cases was wedged but it was just taking a long time because we try O(n^2) different splits.  The actual error messages include the test name, but I still find it helpful to be able to see which test case has started.